app/models/report.rb

```diff
@@ -44,4 +32,20 @@ def unreported_change_types
     reported_change_types = reported_changes.pluck :change_type
     all_change_types - reported_change_types
   end
+
+  def current_change
```

Comment:
I suggest that `current_change` lives in the controller instead of a model. This would be similar to the `current_report` implementation:

https://github.com/codeforamerica/colorado-benefits/blob/3e1ef790960bfa6f9fea439b0b80a898d36ee5ce/app/controllers/forms_controller.rb#L44

I don't think we need to store the ID in the session like for `current_report`, but it does afford us that option in the future if we want to handle edits to changes.

This will likely simplify how `current_change` is called throughout the codebase. We won't have to go through `current_report`.

Reply:
Makes sense, thanks for looking into it. :+1: